alembic/versions/98399f652287_add_books_relationship_to_author.py

```diff
@@ -0,0 +1,31 @@
+"""Add books relationship to Author
+
+Revision ID: 98399f652287
+Revises: 0bdf49a1247b
+Create Date: 2024-10-29 19:43:47.877832
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "98399f652287"
+down_revision: Union[str, None] = "0bdf49a1247b"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function is also empty. Ensure you implement the logic to reverse the changes made in the `upgrade` function, allowing the migration to be rolled back if necessary.